public/js/main.js

```diff
@@ -182,29 +182,39 @@
 
   let pressArray = [
     {
-      date: "December 10, 2019",
-      title: "Lorem Ipsum dolor sit amet, consectetur adipisicing elit.",
-      link: ""
+      date: "June 14, 2019",
```

Comment:
I may be missing context here (as code review only shows changes) but something to think about. Is this date going to come formatted this way or will it come in a raw format (e.g. database timestamp or js datetime.timestamp) that you'll have to convert?

Looks good either way just food for thought.

Reply:
Good point actually. At some point, I'll bring in a date formatter when I use an API to retrieve the data